src/Period.php

```diff
@@ -887,22 +887,37 @@ public function merge(Period $arg)
      * </ul>
      *
      * @param DateInterval|int|string $interval The interval
+     * @param bool                    $backwards Optionnal - iterates counterwise
      *
      * @return Generator|Period[]
      */
-    public function split($interval)
+    public function split($interval, $backwards = false)
```

Comment:
`?bool`? Does this library provide PHP 7.1-only support?

Reply:
Why not a second method `splitBackwards`?

Reply:
@Ocramius this is what I have suggested @bpolaszek to do 👍